mobile_interface.go

```diff
@@ -311,19 +315,25 @@ func OpenDApp(id, context string) error {
 
 }
 
-func StartDApp(dApp string, timeout int) error {
+func StartDApp(dAppSingingKeyStr string, timeout int) error {
 
 	//Exit if not started
 	if panthalassaInstance == nil {
 		return errors.New("you have to start panthalassa first")
 	}
 
-	dAppResp := dapp.JsonRepresentation{}
-	if err := json.Unmarshal([]byte(dApp), &dAppResp); err != nil {
+	// decode singing key
+	dAppSigningKey, err := hex.DecodeString(dAppSingingKeyStr)
+	if err != nil {
 		return err
 	}
 
-	return panthalassaInstance.dAppReg.StartDApp(&dAppResp, time.Second*time.Duration(timeout))
+	// signing key must be 32 bytes long since it's and ed25519 pub key
```

Comment:
Small typo `since it's and` should probably be `since it's an`

Reply:
done